virtcontainers/cgroups.go

```diff
@@ -209,9 +209,9 @@ func (s *Sandbox) deleteCgroups() error {
 }
 
 func (s *Sandbox) constrainHypervisor(cgroup cgroups.Cgroup) error {
-	pid := s.hypervisor.pid()
-	if pid <= 0 {
-		return fmt.Errorf("Invalid hypervisor PID: %d", pid)
+	pids := s.hypervisor.pid()
+	if len(pids) == 0 || pids[0] == 0 {
```

Comment:
But what if `len(pids) > 1 && pid[n] == 0`?

If the code requires that the "main" PID be at index zero, we probably need a comment explaining that (and maybe a unit test? :smile:)